internal/encoding/merge/parquet.go

```diff
@@ -0,0 +1,295 @@
+package merge
+
+import (
```

Comment:
When formatting exports, have an empty line between standard library and custom imports, like so:

```
	"encoding/json"
	"fmt"
	"strconv"
	
	goparquet "github.com/fraugster/parquet-go"
	"github.com/fraugster/parquet-go/parquet"
	"github.com/fraugster/parquet-go/parquetschema"
	"github.com/kelindar/talaria/internal/column"
	"github.com/kelindar/talaria/internal/encoding/block"
	"github.com/kelindar/talaria/internal/encoding/typeof"
	"github.com/kelindar/talaria/internal/monitor/errors"
```